### PR TITLE
Scottx611x/new workflows

### PR DESCRIPTION
**Pull request by scottx611x**
Includes new ATAC-seq workflows and mm9 version of our existing ChIP-seq workflow

@hackdna Could you review and deploy these to the `stemcellcommons` toolshed when you get a chance? 

It would also be great if I could be given the ability to interact with this toolshed account as well.

### CODE REVIEW

**Comment by hackdna** [comment]
What is the difference between `ATAC-seq_-_Mouse.ga` and `ATAC-seq_-_Mouse_(mm9).ga`?

Also, between the new `ChIP-seq_Peak_Calling_-_Mouse_(mm9).ga` and existing `ChIP-seq_Peak_Calling_-_Mouse.ga`?

**Comment by scottx611x** [comment]
@hackdna The non-mm9 ones use the mm10 reference genome. Searching for `refGenomeSource` should clear things up.

**Comment by hackdna** [comment]
I see. Would it be more helpful to indicate this explicitly to the users by including the genome build info in the name (e.g., `ATAC-seq_-_Mouse_(mm10).ga`)?

**Comment by scottx611x** [comment]
I'm not sure, that would be a question for @gmnelson.

**Comment by gmnelson** [comment]
@hackdna My idea was that the logic would be, if no genome version is specified in the workflow name, then the latest available release is being used, and otherwise it is specified. My impression is this will be easier to decipher visually and, in any case, the genome being used is always specified in the workflow description.

At some point not far off, I'd like to generalize the workflows so species/genome build is a selectable parameter, so this would no longer be a concern.

**Comment by hackdna** [comment]
OK, wouldn't it be confusing for the users when a new genome version becomes available but workflows have not been updated yet?

**Comment by gmnelson** [comment]
It's possible, but they can always confirm the build number within the workflow description.

My inclination would be to not worry about changing all the workflow names right now. If a user tells me it confused them, and I'm not close to having more general workflows, then I'd say it's worthwhile to make this change.

File: chipseq_workflows/ChIP-seq_Peak_Calling_-_Mouse_(mm9).ga [hunk 1]
```diff
@@ -0,0 +1,709 @@
...
+            }, 
+            "tool_state": "{\"__page__\": 0, \"__rerun_remap_job_id__\": null, \"input_collection\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", 
+            "tool_version": "0.2", 
+            "type": "tool", 
+            "uuid": "b9706229-8a52-47cc-bb8d-28d8103caae6", 
+            "workflow_outputs": []
+        }, 
+        "2": {
+            "annotation": "", 
+            "content_id": "toolshed.g2.bx.psu.edu/repos/kevyin/fastq_groomer_parallel/fastq_groomer_parallel/0.3.0",
```
Comment (hackdna) [inline chipseq_workflows/ChIP-seq_Peak_Calling_-_Mouse_(mm9).ga]: Version 0.3.0 of `fastq_groomer_parallel` does not match version 0.3.1 for the rest of existing workflows (e.g.: https://github.com/stemcellcommons/scc-galaxy-tools/blob/dc4a5924dc98d22c14eac246506d13dfc13c3c6d/chipseq_workflows/ChIP-seq_Peak_Calling_-_Mouse.ga#L102).

**Review by hackdna — CHANGES_REQUESTED** [review]

**Review by hackdna — APPROVED** [review]

**Pull request merged**